quickjs-libc.c

```diff
@@ -4173,7 +4173,7 @@ static const JSCFunctionListEntry js_os_funcs[] = {
 #if !defined(__wasi__)
     JS_CFUNC_DEF("realpath", 1, js_os_realpath ),
 #endif
-#if !defined(_WIN32) && !defined(__wasi__)
+#if !defined(_WIN32) && !defined(__wasi__) && !defined(_TVOS) && !defined(_WATCHOS)
```

Comment:
It should be possible to make it work without setting these defines externally. Include TargetConditionals.h like this:
```c
#ifdef __APPLE__
#include <TargetConditionals.h>
#endif
```
And then change this line to:
```suggestion
#if !defined(_WIN32) && !defined(__wasi__) && !(defined(__APPLE__) && (TARGET_OS_TV || TARGET_OS_WATCH))
```
Which I won't deny is ugly-looking but :shrug:, as long as it works. There are different ways of writing out the boolean clauses but none are much prettier.